src/test/kotlin/com/terraformation/backend/tracking/model/PlantingZoneModelTest.kt

```diff
@@ -244,24 +236,24 @@ class PlantingZoneModelTest {
                           id = 1,
                           plots =
                               listOf(
-                                  monitoringPlotModel(10, permanentCluster = 1),
-                                  monitoringPlotModel(11, permanentCluster = 1),
-                                  monitoringPlotModel(12, permanentCluster = 1),
-                                  monitoringPlotModel(13, permanentCluster = 1),
-                                  monitoringPlotModel(14, permanentCluster = 2),
-                                  monitoringPlotModel(15, permanentCluster = 2),
-                                  monitoringPlotModel(16, permanentCluster = 2),
-                                  monitoringPlotModel(17, permanentCluster = 2),
+                                  monitoringPlotModel(10, permanentIndex = 1),
+                                  monitoringPlotModel(11, permanentIndex = 1),
+                                  monitoringPlotModel(12, permanentIndex = 1),
+                                  monitoringPlotModel(13, permanentIndex = 1),
+                                  monitoringPlotModel(14, permanentIndex = 2),
+                                  monitoringPlotModel(15, permanentIndex = 2),
+                                  monitoringPlotModel(16, permanentIndex = 2),
+                                  monitoringPlotModel(17, permanentIndex = 2),
```

Comment:
If these are sharing the same `permanentIndex`, doesn't that mean these are clusters?